uavcan_gui_tool/main.py

```diff
@@ -15,10 +15,18 @@
 
 assert sys.version[0] == '3'
 
+from argparse import ArgumentParser
+parser = ArgumentParser(description='UAVCAN GUI tool')
+
+parser.add_argument("--debug", action='store_true', help="enable debugging")
+parser.add_argument("--dsdl", help="path to custom DSDL")
+
+args = parser.parse_args()
+
 #
 # Configuring logging before other packages are imported
 #
-if '--debug' in sys.argv:
+if args.debug:
     sys.argv.remove('--debug')
```

Comment:
This line is no longer needed.

Reply:
you mean remove the sys.argv.remove() line? I can do that if you like, but it may change behavior as sys.argv is still passed to bus_monitor and plotter.

Reply:
Yes, I think it is not really needed here, because the debug flag should propagate to child processes as well. It was a mistake.